client/src/components/reviews/Review.jsx

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
 
-function Review(props) {
+function Review( {review} ) {
   return (
     <div className="review">
-      <div>Individual review tile #1</div>
-      <div>Individual review tile #2</div>
-      <div>Individual review tile #3</div>
+      <div className="review-date">
+        {review.date}
+      </div>
+      <div className="review-stars">
+        {review.rating} stars
+      </div>
+      <div className="review-title">
+        <h3>{review.summary}</h3>
+      </div>
+      <div className="review-body">
+        {review.body}
+      </div>
+      <div className="review-recommend">
+        Recommended? {review.recommend.toString()}
```

Comment:
I'd recommend using a ternary operator here to render **Yes/N**o as opposed to **True/False**

Reply:
@jaimienakayama once you update this, the entire pr is good to merge into dev. nice work! :)